install/installation.md

```diff
@@ -211,7 +211,10 @@ need to be started. This can be done using the next command (assuming 1000 is
 the `gid` and `uid` of the user which was specified earlier):
 
 ```
-sudo pure-uploadscript -u 1000 -g 1000 -B -r /home/path/to/src/cron/upload.sh
+sudo pure-uploadscript -u 2015 -g 2015 -B -r /home/path/to/src/cron/progress_ftp_upload.py
+sudo chown 2015:2015 /home/path/to/src/cron/progress_ftp_upload.py
+sudo chown -R 2015:2015 /repository
```

Comment:
That's a big no no. If this happens, the user of the platform (e.g. www-data) can no longer access it.